src/main/java/net/javadiscord/javabot/command/interfaces/Autocomplete.java

```diff
@@ -0,0 +1,18 @@
+package net.javadiscord.javabot.command.interfaces;
+
+import net.dv8tion.jda.api.events.interaction.command.CommandAutoCompleteInteractionEvent;
+import net.dv8tion.jda.api.requests.restaction.interactions.AutoCompleteCallbackAction;
+
+/**
+ * Interface that handles Discord's Autocomplete functionality.
+ * <p>
+ * To enable this handler for a particular slash command option, navigate to that
+ * option's entry in the corresponding YAML-file, and add the following property:
+ * <pre><code>
+ * autocomplete: true
+ *     </code></pre>
+ * </p>
+ */
+public interface Autocomplete {
```

Comment:
I would actually rename this to something that implies that something can be used for auto-completion instead of something that **is an** `Autocomplete`. An example would be `Autocompletable`

Reply:
Sure